modules/apps/object/object-web/src/main/resources/META-INF/resources/js/utils/fdsSourceDataRenderer.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import classNames from 'classnames';
+import React from 'react';
+
+interface fdsSourceDataRenderer {
+	itemData: {
+		system: boolean;
+	};
+}
+
+export default function fdsSourceDataRenderer({
```

Comment:
Hey @pedro-oliveira446, As it is a React component, you must start the name with a capital letter, in this case you can use `FDSSourceDataRenderer` as the name of this component, rename the file as well please..

Another thing, looking better at the code now, I think we can place this component inside the `FDSPropsTransformer` folder, it makes more sense since `PropsTransformer` and `DataRenders` are basically the same thing.